code-studio/src/js/plc/task_creation.js

```diff
@@ -0,0 +1,28 @@
+/* globals dashboard */
+
+var IconLibrary = require('../components/IconLibrary.jsx');
+
+$(window).load(function () {
+  $('#toggleIconLibrary').click(function () {
+    $('#iconDiv').toggle();
+  });
+
+  $('#unsetIcon').click(function () {
+    $('#plc_learning_resource_task_icon').val('');
+    $('#previewIcon').removeClass();
+    $('#iconDiv').toggle();
+  });
+});
+
+
+ReactDOM.render(React.createElement(IconLibrary, {
+  alignment: 'left',
+  assetChosen: function (name) {
+    var iconClass = name.substring('icon://'.length);
```

Comment:
Let's update the component to return the un-prefixed icon.  In retrospect the consumer should own prefixing the string.